src/globals/scss/_typography.scss

```diff
@@ -65,6 +64,11 @@ $govuk-font-stack-tabular: $govuk-nta-light-tabular !default;
   line-height: (40 / 36);
 }
 
+@mixin govuk-core-27 {
```

Comment:
no, not yet. still assessing which typography styles are being used before surfacing them